### DISCUSSION TITLE
Having a problem on install

### DISCUSSION THREAD

**Discussion by musicjeremy** [backblaze-b2-samples/b2fs4chia#3] [Q&A] [answered]
Hello, I am going through your install process. I hope this is the right place to ask. 

 I have b2fs4chia installed correctly, I believe:

(venv) root@ubuntu-c-2-4gib-nyc1-01:~/b2fs# b2fs4chia --version
b2fs4chia version 0.1.6

(venv) root@ubuntu-c-2-4gib-nyc1-01:~/b2fs# b2fs4chia
usage: b2fs4chia [-h] [--version] [--debug] [--account_id ACCOUNT_ID] [--application_key APPLICATION_KEY] [--bucket_id BUCKET_ID]
                 [--config_filename CONFIG_FILENAME] [--allow_other] [--cache_timeout CACHE_TIMEOUT]
                 mountpoint
b2fs4chia: error: the following arguments are required: mountpoint

But when attempting to mount, I get this error:

(venv) root@ubuntu-c-2-4gib-nyc1-01:~/b2fs# b2fs4chia /mnt/1
/root/b2fs/venv/lib/python3.8/site-packages/b2fuse/b2fuse.py:73: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  return yaml.load(f.read())
Traceback (most recent call last):
  File "/root/b2fs/venv/bin/b2fs4chia", line 8, in <module>
    sys.exit(main())
  File "/root/b2fs/venv/lib/python3.8/site-packages/b2fuse/b2fuse.py", line 102, in main
    config["cacheTimeout"] = 120
TypeError: 'str' object does not support item assignment

Any thoughts or assistance would be greatly appreciated.

**Comment by ppolewicz** [comment]
There is something wrong with your config.yaml, it seems that its a string
and not an object/dict like we expect it to be

**Reply by musicjeremy** [reply]
Hi, thank you for responding. Below are the contents of my config.yaml, nothing else:
accountId:xxxxx
applicationKey:yyyy
bucketId:chia1

**Comment by ppolewicz** [comment, marked as answer]
There is no space after :

**Reply by musicjeremy** [reply]
Thank you, that worked!
